yt_dlp/extractor/common.py

```diff
@@ -2339,7 +2339,7 @@ def _parse_smil_formats_and_subtitles(
         imgs_count = 0
 
         srcs = set()
-        media = smil.findall(self._xpath_ns('.//video', namespace)) + smil.findall(self._xpath_ns('.//audio', namespace))
+        media = [el for arg in ['.//video', './/audio', './/media'] for el in smil.findall(self._xpath_ns(arg, namespace))]
```

Comment:
Mayhaps something like this makes the line easier to understand
```suggestion
        media = itertools.chain.from_iterable(
            smil.findall(self._xpath_ns(arg, namespace))
            for arg in ['.//video', './/audio', './/media'])
```